text/0000-docs-information-architecture

```diff
@@ -0,0 +1,154 @@
+- Start Date: 2018-04-25
+- RFC PR:
+- Gatsby Issue:
+
+# Summary
+
+A proposal to add a second (maybe even third) level of navigation to the docs on gatsbyjs.org and to add new top-level categories.
+
+# Motivation
+
+Through many user research interviews, I've heard that it's difficult for people to find what they are looking for in the docs. They have mentioned many instances where the information they are looking for does not exist, it exists but they couldn't find it, or it exists and they could find it in multiple places (not in one single place). This is likely because the docs have outgrown their current categories.
+This design also proposes a possibly heuristic for determining what we can reasonably commit to maintaining as part of Gatsby core docs, and what might be too far outside the scope of core docs (either because the topics don't fit or because they are not a priority, given our limited time and resources). includes a list of what will *not* be included in Gatsby core docs.
+
+# Detailed design
+
+Here is a possible restructure of the current docs information architecture (with the inclusion of many docs we still need to write):
```

Comment:
Could we mark docs that are not-yet-written with a (*) so it's easy to see what will be re-arranged and what will be created?

Reply:
ok done! Many are not going to be completely from scratch. There is tons of content that needs to be expanded or just reorganized. I starred all docs that either don't exist at all or don't exist in the form I'm envisioning.

Reply:
There are a *few* extra ones I didn't star because even though they don't exist in their current form, it will be as easy as just dividing up a pre-existing doc.